code/core/src/core-server/build-static.ts

```diff
@@ -43,8 +44,12 @@ export async function buildStaticStandalone(options: BuildStaticStandaloneOption
   if (options.outputDir === '/') {
     throw new Error("Won't remove directory '/'. Check your outputDir!");
   }
-  await emptyDir(options.outputDir);
-  await ensureDir(options.outputDir);
+
+  const outputDirFiles = await readdir(options.outputDir);
```

Comment:
what happens if the dir doesn't exist when calling `readdir()`?

Reply:
ah, think you caught the same while I was mid-review.

Reply:
I've got your CI to thank as it's what caught it. 😄